config/vufind/config.ini

```diff
@@ -563,6 +563,22 @@ default_dismax_handler = dismax
 ; memory problems.
 ;record_batch_size = 100
 
+; This section provides settings for optional caching of search requests. Supported
+; for Solr only.
+[IndexCache]
```

Comment:
If we're doing one configuration per index (which I think makes sense, since you might want different caching behavior for different cores), would this make more sense in searches.ini rather than config.ini, since that's a place for Solr-specific settings?

Reply:
Done.

Reply:
Thanks!